packages/ember-intl/addon/services/intl.js

```diff
@@ -91,6 +113,33 @@ const IntlService = Service.extend(Evented, {
     return get(this, 'adapter.seen').map(l => l.localeName);
   }).readOnly(),
 
+  /**
+   * Peeks into the requirejs map and registers all locale data objects found.
+   * This is also very likely to be removed soon.
+   *
+   * @private
+   */
+  _hydrate() {
```

Comment:
This garbage is going away, will be lazily done as part of the addTranslation logic and will go through the owner lookup/resolveRegistration api.